### ISSUE TITLE
Cannot download the data from Google drive, read-only

### ISSUE THREAD

**Issue by Albert-Ma** [xlpczv/Semi_Siamese#2]
Cannot download the data from Google drive, read-only

**Comment by xlpczv** [comment]
Hello, I checked it and found downloading was still possible.
Please try to download it again.

**Comment by Albert-Ma** [comment]
> Hello, I checked it and found downloading was still possible. Please try to download it again.

Hi, google drive says that there is no permission to preview the data and there is no download option.
My teammate also checks this link and still cannot download it.
Maybe you are the owner and have the download  permission.

Just to make sure, did you use the same training set(i.e., official [msmarco-doctrain-top100](https://msmarco.blob.core.windows.net/msmarcoranking/msmarco-doctrain-top100.gz)), dev set ([msmarco-docdev-top100](https://msmarco.blob.core.windows.net/msmarcoranking/msmarco-docdev-top100.gz)) as the official MS MARCO reranking setting. I see there is a 'full.test.run' file but MS MARCO doesn't provide the corresponding qrel file.
And in the README, you say "We used 1024 as 'batches_per_epoch' when training the models using MS-MARCO." and the total training data for one epoch thus in your training is 1024*16/2=8192 queries if you use 1 negative document for a query and 16 as the training batch size.
So I was confused as the MS MARCO document has 0.5M queries and there should be 62.5k steps when the batch size is 16.

**Comment by xlpczv** [comment]
Hi, I checked the download link again. This time it will work. Thanks for noting me.

Answers to your questions are followed.
We used the official MS-MARCO dataset, but the dataset we used is not the most recent one for now. We used the label information provided on the homepage after the contest ended, so we had qrels. For now, it seems that they don't provide the label data any more. We recommend you use the latest data.
Actually, the model doesn't go through all the training queries per each epoch. However, with the default arguments for max_epoch, batch_size, and batches_per_epoch, the model sees all queries at least once.
For processing the data, we followed the CEDR paper ([17] in our paper). For more details, please refer to the experiment section of our paper or the CEDR paper.

**Comment by Albert-Ma** [comment]
Thanks, the data can be downloaded now.

Can you provide the reranking result of MS MARCO?  Since I want to add your method as a baseline, I need the performance in terms of MRR@10 and MRR@100. 
That would be very helpful if you can provide either the reranking results or the values of MRR@10 and MRR@100.

**Comment by xlpczv** [comment]
I replied to the mail. Please check a mail box.

**Comment by soroushjavdan** [comment]
Hello, thank you for sharing the code. I wonder if you could re generate the dataset link. Unfortunately, It is not working.

**Comment by xlpczv** [comment]
Thank you for letting me know the problem. The link is available now.
